SConstruct

```diff
@@ -630,7 +630,13 @@ if env["scu_build"]:
 # are actually handled to change compile options, etc.
 detect.configure(env)
 
-print(f'Building for platform "{env["platform"]}", architecture "{env["arch"]}", target "{env["target"]}".')
+if env.get("ios_simulator") or env.get("visionos_simulator"):
+    platform_string = env["platform"] + " (simulator)"
+else:
+    platform_string = env["platform"]
```

Comment:
I think this could be simplified to just:
```suggestion
platform_string = env["platform"]

if env.get("ios_simulator") or env.get("visionos_simulator"):
    platform_string += " (simulator)"
```